index.md

```diff
@@ -5,6 +5,17 @@ title: "Ruby Unconf Hamburg 2019"
 
 <div class="content-section content-section--whitebg" markdown="1">
 
+## Tickets
```

Comment:
maybe below the 2019 section because the users should read the basic facts first?